src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person?.name}</h2>
+
+    {person?.age !== undefined && person?.age !== 0 && (
+      <p className="Person__age">I am {person?.age}</p>
+    )}
+
+    <p className="Person__partner">
+      {person?.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+        : 'I am not married'}
```

Comment:
Consider adding a check for `partnerName` to ensure it is defined before using it in the template. This will prevent potential issues if `partnerName` is missing for married individuals.